environment/src/main/resources/application.yml

```diff
@@ -93,6 +93,10 @@ environment:
   sdx:
     url: http://localhost:8086
     contextPath: /dl
+  consumption:
+    url: http://localhost:8099
+    contextPath: /consumption
+    enabled: false
```

Comment:
Is it intentionally disabled?

Reply:
Thanks for clarifying, it was my guess too that it's mainly for local setups, just wanted to be sure.